diff --git a/node_modules/dero-xswd-api/dist/index.d.ts b/node_modules/dero-xswd-api/dist/index.d.ts
index ca42002..aa187be 100644
--- a/node_modules/dero-xswd-api/dist/index.d.ts
+++ b/node_modules/dero-xswd-api/dist/index.d.ts
@@ -217,7 +217,7 @@ type Network = "Simulator" | "Testnet" | "Mainnet";
 type Status = {
     status: "OK";
 };
-type DEROGetInfoResult = {
+export type DEROGetInfoResult = {
     alt_blocks_count: Uint64;
     difficulty: Uint64;
     grey_peerlist_size: number;
@@ -484,7 +484,7 @@ declare class Api {
         GetLastBlockHeader(): Promise<Response<"daemon", "DERO.GetLastBlockHeader", "error"> | Response<"daemon", "DERO.GetLastBlockHeader", "result">>;
         GetBlockTemplate(params: DEROGetBlockTemplate): Promise<Response<"daemon", "DERO.GetBlockTemplate", "error"> | Response<"daemon", "DERO.GetBlockTemplate", "result">>;
         GetEncryptedBalance(params: DEROGetEncryptedBalance): Promise<Response<"daemon", "DERO.GetEncryptedBalance", "error"> | Response<"daemon", "DERO.GetEncryptedBalance", "result">>;
-        GetSC(params: DEROGetSC, waitAfterNewBlock?: true): Promise<Response<"daemon", "DERO.GetSC", "error"> | Response<"daemon", "DERO.GetSC", "result">>;
+        GetSC(params: DEROGetSC, waitAfterNewBlock?: boolean): Promise<Response<"daemon", "DERO.GetSC", "error"> | Response<"daemon", "DERO.GetSC", "result">>;
         GetGasEstimate(params: DEROGetGasEstimate): Promise<Response<"daemon", "DERO.GetGasEstimate", "error"> | Response<"daemon", "DERO.GetGasEstimate", "result">>;
         NameToAddress(params: DERONameToAddress): Promise<Response<"daemon", "DERO.NameToAddress", "error"> | Response<"daemon", "DERO.NameToAddress", "result">>;
     };
